lib/cassia/access_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Cassia
+  class AccessController
+    include Virtus.model
+
+    attribute :access_token, String, default: nil
+    attribute :error, String, default: nil
+    attribute :error_description, String, default: nil
+    attribute :autoselect_switch, Integer, default: 0
+    attribute :device_mac, String, default: nil
```

Comment:
I think instead of calling this `device_mac` we should have a `connected_devices` array which uses a `Device` model. Then the device model has a `mac` attribute on it that stores the mac. So basically when you connect to a new device you create a new `device` object and push it into an array of `connected_devices`. The default would be an empty array, something like this https://github.com/rocketsofawesome/newgistics-ruby/blob/master/lib/newgistics/order.rb#L20 and https://github.com/rocketsofawesome/newgistics-ruby/blob/master/lib/newgistics/item.rb